src/components/ProfilePage.js

```diff
@@ -1,20 +1,63 @@
-import React, {useState} from 'react';
+import React, {useContext, useEffect, useState} from 'react';
 import './ProfilePage.css';
 import { Container, Form, Button, Row, Col} from 'react-bootstrap';
-import picLogScreen from '../assets/picLogScreen.png'
+import Navbar from "./Navbar";
+import {UserContext} from '../util/context';
+import { axiosInstance } from '../util/config';
 
 export default function ProfilePage() {
+
+    const [error, setError] = useState(false);
+    const [errorMessage, setErrorMessage] = useState('');
+    const user = JSON.parse(window.localStorage.getItem('username'));
```

Comment:
Instead of accessing the username through localStorage in this file, it can be accessed in the wrapper and passed down through the context.